pulsar-io/elastic-search/src/test/java/org/apache/pulsar/io/elasticsearch/testcontainers/ChaosContainer.java

```diff
@@ -19,26 +19,63 @@
 package org.apache.pulsar.io.elasticsearch.testcontainers;
 
 import lombok.extern.slf4j.Slf4j;
+import org.awaitility.Awaitility;
 import org.testcontainers.containers.BindMode;
 import org.testcontainers.containers.GenericContainer;
 import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.containers.wait.strategy.WaitStrategy;
 
+import java.util.ArrayList;
+import java.util.List;
 import java.util.Optional;
+import java.util.concurrent.TimeUnit;
+import java.util.function.Consumer;
+import java.util.function.Predicate;
 
 // see https://github.com/alexei-led/pumba
 @Slf4j
 public class ChaosContainer<SELF extends ChaosContainer<SELF>> extends GenericContainer<SELF> {
 
-  public static final String PUMBA_IMAGE = Optional.ofNullable(System.getenv("PUMBA_IMAGE"))
-          .orElse("gaiaadm/pumba:latest");
-
-  public ChaosContainer(String targetContainer, String pause) {
-    super(PUMBA_IMAGE);
-    setCommand("--log-level info --interval 60s pause --duration " + pause + " " + targetContainer);
-    addFileSystemBind("/var/run/docker.sock", "/var/run/docker.sock", BindMode.READ_WRITE);
-    setWaitStrategy(Wait.forLogMessage(".*pausing container.*", 1));
-    withLogConsumer(o -> {
-      log.info("pumba> {}", o.getUtf8String());
-    });
-  }
+    public static final String PUMBA_IMAGE = Optional.ofNullable(System.getenv("PUMBA_IMAGE"))
+            .orElse("gaiaadm/pumba:latest");
```

Comment:
Are we sure that we want 'latest'?
This means that the test is not reproducible and it will probably fail some day